package.json

```diff
@@ -34,7 +34,7 @@
     "@docusaurus/preset-classic": "2.4.1",
     "@mdx-js/react": "1.6.22",
     "clsx": "1.2.1",
-    "docusaurus-lunr-search": "2.3.2",
+    "docusaurus-lunr-search": "2.4.1",
```

Comment:
❓ I looked at release notes here and didn't see why the search bar theme would need to be deleted. Is this just coincidence?

Reply:
They had some changes to their templates and had updated their readme to not suggest "ejecting" the templates. I verified that the search bar worked in both light and dark theme without the template overrides so it seemed safe to remove them instead of re-ejecting them.